server/src/main/java/com/objectcomputing/checkins/services/role/member_roles/MemberRoleServicesImpl.java

```diff
@@ -39,6 +39,10 @@ public Optional<MemberRole> findById(MemberRoleId memberRoleId) {
         return memberRoleRepository.findById(memberRoleId);
     }
 
+    public Optional<MemberRole> findByMemberId(UUID memberId) {
+        return memberRoleRepository.findByMemberId(memberId);
+    }
+
```

Comment:
I'm not entirely sure why this was added, as it's not being used anywhere. I'm guessing it may have been an alternative approach to calling `roleServices.findUserRoles(id)` in `RolePermissionServicesImpl::findUserPermissions`